lib/utils.js

```diff
@@ -324,7 +347,7 @@ export function moveElementAwayFromCollision(layout: Layout, collidesWith: Layou
     // Make a mock item so we don't modify the item here, only modify in moveElement.
     const fakeItem: LayoutItem = {
       x: compactH  ? Math.max(collidesWith.x - itemToMove.w, 0) : itemToMove.x,
-      y: !compactH ? Math.max(collidesWith.y - itemToMove.h, 0) : itemToMove.y,
+      y: itemToMove.y,
```

Comment:
Can you explain why this was changed - and not changed for `x`?

Reply:
Yea, think both might be wrong. This was changed when horizontal compaction was introduced, not sure why the compactH ?  condition was introduced. so left it for horizontal compaction as I have not tested that.

https://github.com/STRML/react-grid-layout/commit/9b1260d4c41eb900a1e853277c4711114369de0b#diff-50e3aa130a4f97a42ee2cf111c7b1d9dL301

I can test horizontal compaction and update PR so that it address issues for that mode as well.

Reply:
It very well may be. That would be very helpful if you could give it a look and ensure horizontal compaction works as expected.